pkg/models/results.go

```diff
@@ -82,6 +85,27 @@ func (groupInfo *GroupInfo) IsCalled() bool {
 	return false
 }
 
+func (groupInfo *GroupInfo) IndexString() string {
+	// Assumes IDs is sorted
+	return strings.Join(groupInfo.IDs, ",")
+}
+
+// IsFixed returns a list of fixed versions, or an empty slice if no fixed version is available
+func (vuln *Vulnerability) FixedVersions() []string {
```

Comment:
Should we be matching on the exact package name? Given that an OSV can have multiple packages.

Reply:
Good point, fixed now and I added a unit test to the grouping function that uses this function.